compiler/rustc_resolve/src/lib.rs

```diff
@@ -1073,6 +1073,8 @@ pub struct Resolver<'ra, 'tcx> {
     module_children: LocalDefIdMap<Vec<ModChild>>,
     trait_map: NodeMap<Vec<TraitCandidate>>,
 
+    typeck_children: LocalDefIdMap<Vec<LocalDefId>>,
```

Comment:
some comment about how this is just the syntactical typeck children collected after expansion, and does not consider synthetic defs at all.